scrooge-generator/src/test/scala/com/twitter/scrooge/ThriftStructMetaDataSpec.scala

```diff
@@ -162,7 +162,7 @@ class ThriftStructMetaDataSpec extends Spec {
     val bonk = MorePerfectUnion.Bonk(Bonk("message", 55))
     assert(Some(bonkFieldInfo) == bonk.unionStructFieldInfo)
 
-    bonk.containedValue match {
+    bonk.containedValue() match {
```

Comment:
Quite correct - have fixed now and is ready for checking as soon as the builds are done.